docs/application/dotnet/guides/nui/nui-components/Notification.md

```diff
@@ -0,0 +1,52 @@
+# Notification
+Notification is a common component that is help to raise a notification window with a content View.
+
+A Notification can be created using property.
+
+![Notification](./media/notification.png)
+
+## Create with Property

```

Comment:
Please use sentene case for heading

## Create with property

Reply:
Done